user-manuals/users/settings.md

```diff
@@ -0,0 +1,7 @@
+## Settings
+
+#### Timezone
```

Comment:
The page title should be in h1 (single #) and both of the sections directly below it should been h2 (two ##s)

Reply:
Edited, please check.